### ISSUE TITLE
Post Scheduling doesn't work when running Ghost on a path

### ISSUE THREAD

**Issue by marcuspoehls** [TryGhost/Ghost#7388]
### Issue Summary

We're running Ghost on a path ([futurestud.io/tutorials](https://futurestud.io/tutorials)) and scheduling posts doesn't work, because the url doesn't get resolved correctly. We defined `{ url: 'http://futurestud.io/blog' }` in our `config.js` for the base url and handle SSL using nginx.

Ultimately, the post scheduler resolves the base url and scheduling url to 

```
https://futurestud.io/blog/blog/ghost/api/v0.1
```

instead of 

```
https://futurestud.io/blog/ghost/api/v0.1
```

See the duplicated `/blog/blog` in the first url?

Thanks for your help!

### Steps to Reproduce

1. Update your Ghost instance to at least version `0.9` (the first version shipping post scheduling)
2. Create a post and schedule it
3. See log output with `PUT` request from scheduler that should release the post on time
4. The log output contains the wrong url

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

### Technical details:

* Ghost Version: `0.10.1`
* Node Version: `4.5.0` 
* Browser/OS: -
* Database: MySQL

**Comment by ErisDS** [comment]
@marcuspoehls Thank you so much for the detailed bug report. It makes our lives sooooo much easier when we get these! 🍰 

You caught us seconds before cutting the next release, but as this bug is pretty serious & you've already helped us debug it, we're going to squeeze a fix in. Hang tight!

**Comment by marcuspoehls** [comment]
@ErisDS @kirrg001 Friends, thank you so much!

**Comment by ErisDS** [comment]
This was shipped yesterday in 0.11.0, just in case you missed it :)

**Comment by marcuspoehls** [comment]
Roger that! Thanks for letting us know :)
